setup.py

```diff
@@ -2,8 +2,11 @@
 
 from setuptools import setup
 from setuptools import find_packages
+import sys
 import re
 
+if sys.version_info < (3, 6):
```

Comment:
in order to reduce the future maintainance hassle, please change this to specifying the minimum python version in a single place, i.e.:
```python
min_python_version = (3,6)
min_python_version_string = ".".join(map(str, min_python_version))
if sys.version_info < min_python_version:
  sys.exit(f'Sorry, Python < {min_python_version_string} is not supported')
# ...
setup(name='cantools',
      python_requires=f'>={min_python_version_string}'
      # ...
      )
```

Reply:
(interestingly enough, this even works with python 2: In the past, I occasionally ran `setup.py` using python 2.7 by mistake and got very non-intuitive error messages.)